epub33/rs/index.html

```diff
@@ -475,7 +475,7 @@ <h3>Manifest</h3>
 						id="confreq-rs-pkg-manifest-unknown" data-tests="#pkg-manifest-unknown">Reading Systems MUST
 						ignore values of the <code>properties</code> attribute they do not recognize.</span></p>
 
-				<p id="confreq-rendition-rs-manifest" data-tests="#pkg-manifest-unlisted-resource">It SHOULD NOT use
+				<p id="confreq-rendition-rs-manifest" data-tests="#pkg-manifest-unlisted-resource">Reading Systems SHOULD NOT use
```

Comment:
Were lines 1268-86 supposed to be committed?

Reply:
I do not understand your question @dlazin